maxfw/utils/image_functions.py

```diff
@@ -155,10 +158,10 @@ def standardize(img, mean=None, std=None):
             mean = np.array([mean] * channels).reshape((1, 1, channels))
         elif isinstance(mean, Sequence):
             # convert a sequence to the right dimensions
-            assert np.sum([not isinstance(x, (int, float)) for x in mean]) == 0, \
-                'The sequence `mean` can only contain numbers.'
-            assert len(mean) == channels, \
-                'The size of the `mean` array must correspond to the number of channels in the image.'
+            if np.sum([not isinstance(x, (int, float)) for x in mean]) != 0:
```

Comment:
Since `mean` can be relatively large, a quicker way might be `np.issubdtype(mean.dtype, np.number)`.

Reply:
```suggestion
            if np.issubdtype(mean.dtype, np.number):
```

Reply:
Should we revert this?

```
File "/home/travis/build/IBM/MAX-Framework/maxfw/utils/image_functions.py", line 161, in standardize
if np.issubdtype(mean.dtype, np.number):
AttributeError: 'list' object has no attribute 'dtype'
```

Reply:
Hmm, `mean` can be a list. OK, I guess we don't need `np.sum` then:

    if any(not isinstance(x, (int, float)) for x in mean):